### PR TITLE
[1/2] Prepare for Marketplace

### PR DESCRIPTION
**Pull request by nathanaelhoun**
Related to #30

### CODE REVIEW

**Comment by darkLord19** [comment]
I think we add only admin config part in how to. And Add a section for How To setup(for admin of server) and link it to howto.md and below that add a section for features and user documentation named" How to Use" or "Features" for slash commands info. That seems better because most people would want to know that details so makes sense to put it on front page.

**Comment by darkLord19** [comment]
I am on phone now so, mistakenly edited your reply. I'm not sure how I was allowed to do that! Anyway, I'll do the changes tomorrow. @nathanaelhoun

**Comment by nathanaelhoun** [comment]
Oopsi closed by mistake. So we are even now xD
You are a full maintainer of this repo so you have all the rights ^^

**Comment by darkLord19** [comment]
@nathanaelhoun I had some time to think about it and it seems that keeping everything in readme seems better to me

**Comment by nathanaelhoun** [comment]
@darkLord19 You are right, it's way better like this. I have updated the links in the docs and in the plugin code. Just have to [peek](https://github.com/phw/peek) some screenshot and we'll be good with this PR :tada:

File: README.md [hunk 1]
```diff
@@ -1,70 +1,191 @@
...
+## Prerequisites
 
-### Manage CircleCI projects
+This guide assumes you have:
 
-  * `/circleci project list-followed` — List followed projects
-  * `/circleci project recent-build <branch>` — List the 10 last builds for a project
-  * `/circleci project env <list|add|add>` — get, add or remove environment variables for given project
+-   A project, hosted on Github or Bitbucket,
+-   A CircleCI account, which can access to the project,
```
Comment (darkLord19) [inline README.md]:
```suggestion
-   A CircleCI account, which has access to the projects/org you want to interact with,
```

**Review by darkLord19 — APPROVED** [review]

**Pull request merged**